### ISSUE TITLE
Thor damage

### ISSUE THREAD

**Issue by luizxd0** [DragonBoundR/DragonBound#29]
When using adukka T2, it does increases Thor level but the damage doesnt change. It doesnt add damage to shots.

**Comment by Ox18** [comment]
The thor bullet makes a hole but does not generate damage? is that what you mean?
I'll work on this first, then we can continue with bullets and climates.

**Comment by luizxd0** [comment]
Yes. It creates the hole but doesn't increase damage. It was supposed to increase damage based on the points/level.
